### ISSUE TITLE
Repeating madx program

### ISSUE THREAD

**Issue by coldfix** [MethodicalAcceleratorDesign/MAD-X#360]
*Issue migrated from trac ticket # 360*

**component:** _build | **priority:** major | **resolution:** wontfix
___
#### 2015-08-14 09:52:06: masamitsu.aiba@psi.ch commented

This is unusual usage of Madx but using the methods defined in mad_core.h
When Madx is called from C code and closed,
no command is accepted in the second run.
___
#### 2015-08-14 09:52:14: alatina uploaded file [`Case3.c`](https://raw.githubusercontent.com/MethodicalAcceleratorDesign/madx-tickets/master/attachments/360/Case3.c) (0.2 KiB)

___
#### 2015-09-11 16:13:38: @ldeniau commented

date: 2015.09.11, from ldeniau
We do not support the use of MAD-X as a library, but we do provide it at your own risk because using MAD-X at this level requires a deep knowledge of the code. Moreover, MAD-X was originally developed with the one-go in mind, so restarting it after a close is absolutely not supported. The new MAD (foreseen for 2017) will have the notion of workspace (i.e. environment) and will allow to run few MAD in parallel or close and restart a workspace at will.

___
#### 2015-09-11 16:13:38: @ldeniau set owner to *ldeniau*

___
#### 2015-09-11 16:13:38: @ldeniau changed status from *new* to *assigned*

___
#### 2015-09-11 16:13:56: @ldeniau set resolution to *wontfix*

___
#### 2015-09-11 16:13:56: @ldeniau changed status from *assigned* to *closed*
